### DISCUSSION TITLE
Quanto tempo de vida terá os novos frameworks?!

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#714] [Discussão]
Nessa "briga" dos frameworks modernos vemos todo dia nascer conceitos novos e novas formas de resolver problemas, mas fica a dúvida, quanto tempo de vida terá esses novos frameworks (angular x, vuejs, react)?

**Comment by kvnol** [comment]
Tem que ver o que cada um vê como "tempo de vida", pois para mim se o framework/ferramenta ainda tem um core team trazendo releases, o mesmo ainda está vivo.

Diferentemente de hype, porque tudo sai de hype uma hora e isso não diz nada sobre a ferramenta ter ficado ruim ou o novo hype ser melhor. Apenas está no hype.

**Comment by mksanti** [comment]
Concordo com o @kvnol. Em algum momento o backbone ou jquery por exemplo ficaram na hype. 
Mas até hoje continuam recebendo atualizações.

![image](https://user-images.githubusercontent.com/5166571/30116308-8f3950b8-92f3-11e7-9d32-cfac73b61edb.png)

![image](https://user-images.githubusercontent.com/5166571/30116319-96378e66-92f3-11e7-9e39-515d08f35a7e.png)

**Comment by calaca** [comment]
Essa questão envolve vários fatores como o apontado acima de hype _vs_ suporte. Acho que um lugar pra ter uma ideia legal de como andam os frameworks JS seria [nessa pesquisa de 2016](http://stateofjs.com/2016/introduction/).

**Comment by felipefialho** [comment]
Não acho que é possível fazer prognosticos nesse sentido.

jQuery foi lançado faz quase 10 anos e ainda é a lib mais usada no mundo.

**Comment by felipeuntill** [comment]
Primeiramente gostaria de salientar que os 3 frameworks que você citou estão todos consolidados, absolvidos pela comunidade e grandes empresas, O unico dos citados que chega perto de ser um hype é o Vue porém com uma breve consultada no github deles vemos que o framework vem em constante evolução desde [2013 ](https://github.com/vuejs/vue/tags?after=v0.7.6).

O mercado Javascript muda mais nem tanto assim amigo, [React](https://github.com/facebook/react/releases/tag/v0.3.0) e [Vue ](https://github.com/vuejs/vue/releases/tag/0.6.0) são frameworks de meados de 2013 e o [Angular ](https://github.com/angular/angular.js/releases/tag/v0.9.3) foi lançado em 2010. 

Frameworks são criadas para resolver problemas, se alguma delas resolver o seu, não hesite em utilizar mesmo que em alguns dos casos você bata cabeça. _Melhor ser ousado que defasado...._
